locust/main.py

```diff
@@ -115,6 +116,15 @@ def parse_options():
         help="Port that locust master should bind to. Only used when running with --master. Defaults to 5557. Note that Locust will also use this port + 1, so by default the master node will bind to 5557 and 5558."
     )
 
+    parser.add_option(
+        '--expect-slaves',
```

Comment:
In that case I think i'd still prefer `--num-slaves` or `--expected-slave-count`. Though it might be bikeshedding, and I'm fine with `--min-slaves` as well :).

> but later realized I need 11 slaves to reach my desired throughput, then I could add another 1 slave after the original 10 slaves have already started swarming

That would require us to have some kind of automatic re-balancing of simulated users when new slaves connect.